### PR TITLE
fix: [PL-17678]: Fix the issue where we are returning the status of same f…

### PR DESCRIPTION
**Pull request by DeepakPatankar**
…ile as success and failure"

Context: 
When we are processing some yamls through api, we return the processing status of each file. The expected response would be of type

 fileName: file1.yaml
 status: SUCCESS

 fileName: file2yaml
 status: SUCCESS

 fileName: file3.yaml
 status: FAILURE

 fileName: file4.yaml
 status: FAILURE
 
 
 
 Because of our bug we were showing the failed files twice, once with the status SUCCESS and once with status FAILED

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/32240)

### CODE REVIEW

**Review by vardanbansal-harness — APPROVED** [review]
LGTM. Add a UT is possible around this change.

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32240#-:-N-YBPpS2x2NoFiC3i8c:b-jgh0g2)* status: :shipit: complete! ✓

File: 400-rest/src/main/java/software/wings/service/impl/yaml/service/YamlServiceImpl.java [hunk 1]
```diff
@@ -1220,12 +1222,30 @@ private YamlOperationResponse prepareFailedYAMLOperationResponse(final String er
     final List<Change> failedChangeList =
         new LinkedList<>(processingFailures.values()).stream().map(value -> value.getChange()).collect(toList());
     final List<Change> processedChangeList =
```
Comment (abhinav-harness) [inline 400-rest/src/main/java/software/wings/service/impl/yaml/service/YamlServiceImpl.java:1224]: looks like this is incorrect variable name - should be succsfull

**Review by DeepakPatankar — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32240)* status: Some of the checks are too old. 1 file(s) to review. 1. unresolved discussions. 1 of 2 approvals obtained. 

___
*[400-rest/src/main/java/software/wings/service/impl/yaml/service/YamlServiceImpl.java, line 1224 at r1](https://reviewable.io/reviews/harness/harness-core/32240#-N-YAscFWB18WcDsAmzs-r1-1224:-N-vBt_lCBBQrgF4z5eT:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/1fef3689e1e69bfc199c250287b50c3e3c3cfa74/400-rest/src/main/java/software/wings/service/impl/yaml/service/YamlServiceImpl.java#L1224)):*
<details><summary><i>Previously, abhinav-harness (Abhinav Singh) wrote…</i></summary><blockquote>

looks like this is incorrect variable name - should be succsfull 
</blockquote></details>

Done.

**Comment by DeepakPatankar** [comment]
trigger message-check

**Comment by DeepakPatankar** [comment]
trigger message_metadata

**Comment by DeepakPatankar** [comment]
trigger message_metadata

**Comment by DeepakPatankar** [comment]
trigger messagecheck

File: 400-rest/src/main/java/software/wings/service/impl/yaml/service/YamlServiceImpl.java (hunk 1 above)
Comment (DeepakPatankar) [reply 400-rest/src/main/java/software/wings/service/impl/yaml/service/YamlServiceImpl.java]: done

**Review by abhinav-harness — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32240#-:-N-w6-7W-cou50t-DlEF:bnfp4nl)* status: 2 file(s) to review. 1 of 2 approvals obtained.

**Review by abhinav-harness — APPROVED** [review]
> Reviewed 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32240)* status: 2 file(s) to review. 1 of 2 approvals obtained.

**Review by abhinav-harness — APPROVED** [review]
> Reviewed 1 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32240)* status: 2 file(s) to review. 1 of 2 approvals obtained.

**Review by abhinav-harness — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32240#-:-N-w6LUhEjVITfWnLBXZ:bnfp4nl)* status: 2 file(s) to review. 1 of 2 approvals obtained.

**Review by rathodmeetsatish — APPROVED** [review]
> Reviewed 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32240)* status: :shipit: complete! ✓

**Pull request merged**